src/components/votersDisplay/container/votersDisplayContainer.js

```diff
@@ -25,24 +25,9 @@ class VotersDisplayContainer extends PureComponent {
     this.state = {};
   }
 
-  // Component Life Cycle Functions
-
-  // Component Functions
-  _handleOnUserPress = (username) => {
-    const { navigation } = this.props;
-
-    navigation.navigate({
-      routeName: ROUTES.SCREENS.PROFILE,
-      params: {
-        username,
-      },
-      key: username,
-    });
-  };
-
   render() {
-    return <VotersDisplayView handleOnUserPress={this._handleOnUserPress} {...this.props} />;
+    return <VotersDisplayView {...this.props} />;
```

Comment:
we are moving unnecessary props to view with {...this.props}, I think it will better if we move just needed props